datastore/db/metadata/picture/__init__.py

```diff
@@ -61,8 +61,8 @@ def build_picture(
     )
     try:
         validator.ProcessedPicture(**picture.model_dump())
-    except Exception:
-        raise PictureCreationError("Error: Picture not created") from None
+    except validator.ValidationError as e:
+        raise PictureCreationError(f"Error, Picture not created:"+str(e)) from None
```

Comment:
Probably for next issue, but avoid using `str(e)` use `e.arg[0]` instead

Reply:
IDK, I've followed the doc? https://docs.pydantic.dev/latest/errors/errors/